src/components/nav-links.tsx

```diff
@@ -17,13 +18,19 @@ export function NavLinks({
     icon: LucideIcon;
   }[];
 }) {
+  const currentPath = usePathname();
   return (
-    <SidebarGroup className="group-data-[collapsible=icon]:hidden">
+    <SidebarGroup>
       <SidebarMenu>
         {links.map((item) => (
           <SidebarMenuItem key={item.name}>
             <SidebarMenuButton asChild>
-              <a href={item.url}>
+              <a
+                href={item.url}
+                className={`flex items-center space-x-2 rounded-md p-2 ${
+                  currentPath === item.url ? "bg-gray-200" : ""
+                }`}
+              >
```

Comment:
@tharcissie better to use [Link ](https://nextjs.org/docs/pages/api-reference/components/link) component form next.js to get all benefits on it instead of norm <a> HTML element